lecture_2/homework/app/controllers/warriors_controller.rb

```diff
@@ -0,0 +1,47 @@
+class WarriorsController < ApplicationController
+  def index
+    if params.has_key?(:alive)
+      warrior = clan.warriors.where(died: nil)
+    elsif params.has_key?(:dead)
+      warrior = clan.warriors.where.not(died: nil)
+    else
+      warrior = clan.warriors.all
+    end
+    render json: warrior
+  end
+
+  def show
+    render json: warrior, include: 'clan', include: 'defensible' ,include: 'weapon'
```

Comment:
To niestety nie zadziała. `render` jest metodą jak wszystkie inne, więc kolejne `include` będą nadpisywać poprzednie. Powinieneś wszystkie te modele zwinąć do tablicy